assets/section-multicolumn.css

```diff
@@ -109,10 +109,6 @@
     padding-left: 1.5rem;
     padding-right: 1.5rem;
   }
-
-  .multicolumn-list.slider .multicolumn-list__item {
-    width: calc(100% - 3rem);
-  }
 }
```

Comment:
Interesting 🤔 I wonder if this was added for a specific reason. But I agree that since this is already on main we can fix it in a follow up. I believe @kmeleta worked on multicolumn, maybe you have more context on this?